components/library/libcouchbase/manifests/sample-manifest.p5m

```diff
@@ -109,18 +109,14 @@ file path=usr/include/libcouchbase/vbucket.h
 file path=usr/include/libcouchbase/views.h
 file path=usr/include/libcouchbase/visibility.h
 link path=usr/lib/$(MACH64)/libcouchbase.so target=libcouchbase.so.2
-link path=usr/lib/$(MACH64)/libcouchbase.so.2 target=libcouchbase.so.2.0.36
-file path=usr/lib/$(MACH64)/libcouchbase.so.2.0.34
-file path=usr/lib/$(MACH64)/libcouchbase.so.2.0.35
-file path=usr/lib/$(MACH64)/libcouchbase.so.2.0.36
```

Comment:
@xen0l seems like it includes old objects too. This is not the same sample-manifest you would generate on clean environment